docs/ref/django-admin.txt

```diff
@@ -1051,6 +1051,20 @@ Mails the email addresses specified in :setting:`ADMINS` using
 
 Starts the Python interactive interpreter.
 
+All models from all installed apps are automatically imported. See the guide on
+:ref:`customizing this behaviour <customizing-shell-auto-imports>` to add or
+remove imports.
+
+Models from apps listed earlier in :setting:`INSTALLED_APPS` take precedence.
+Access models from later, lower-precedence apps through its models module,
+which is made available as ``<app_label>_models``, for example
```

Comment:
```suggestion
You can access overridden models from later, lower-precedence apps through its
models module, which is made available as ``<app_label>_models``, for example
```
We lost the clarification that these are models with the same name, adding "overridden" to try and emphasize this